step-by-step/04-check-compat.md

```diff
@@ -1,24 +1,20 @@
 # 4. Checking compatibility
 
-We are going to check if this device is supported.
+We are going to check if your device is supported.
 
-1. launch RVM. Ignore any updates it claimed it knew.
+1. Launch ReVanced Manager. Ignore any updates it claimed it knew.
 ==- Image for reference
 ![launch rvm](https://github.com/SodaWithoutSparkles/ReVanced-troubleshooting-guide/blob/main/screenshots/040-first_launch_manager.jpg?raw=true)
 ===
 2. Go to the settings tab and scroll down until you find "About"
 ==- Image for reference
 ![check about](https://github.com/SodaWithoutSparkles/ReVanced-troubleshooting-guide/blob/main/screenshots/050-check_about.jpg?raw=true)
 ===
-3. Check that `Arch` had at least 1 entry that looks like `arm64-v8a`
-4. If you only got something that said `armv7` or lower, you cannot patch it on your phone. Please patch it on another device or your PC.
-5. If you got `armv8` or higher, you are probably fine
-6. Check that your Android version is at least 8
-7. If you only got something that said `7` or lower, you cannot patch it on your phone. Please patch it on another device or your PC.
-8. If you made it here, you can probably patch on your phone with ReVanced manager
```

Comment:
I'm pretty sure the Manager won't be able to be installed at all on android 7 and below. I asked kitadai on discord to verify. Waiting to hear back

If this is true, a troubleshooting tip must be added to step 3 for users who are unable to install the manager